.github/workflows/scripts/e2e/gmc_test.sh

```diff
@@ -145,6 +151,53 @@ function validate_codegen() {
    fi
 }
 
+function validate_codetrans() {
+       # todo select gaudi or xeon
+   kubectl create ns $CODETRANS_NAMESPACE
+   sed -i "s|namespace: codetrans|namespace: $CODETRANS_NAMESPACE|g"  $(pwd)/config/samples/codetrans.yaml
+   kubectl apply -f $(pwd)/config/samples/codetrans.yaml
+
+   # Wait until the router service is ready
+   echo "Waiting for the codetrans router service to be ready..."
+   wait_until_pod_ready "codetrans router" $CODETRANS_NAMESPACE "router-service"
+   output=$(kubectl get pods -n $CODETRANS_NAMESPACE)
+   echo $output
+
+
+   # deploy client pod for testing
+   kubectl create deployment client-test -n $CODETRANS_NAMESPACE --image=python:3.8.13 -- sleep infinity
+
+   # wait for client pod ready
+   wait_until_pod_ready "client-test" $CODETRANS_NAMESPACE "client-test"
+   # giving time to populating data
+   sleep 60
+
+   kubectl get pods -n $CODETRANS_NAMESPACE
+   # send request to codetrans
+   export CLIENT_POD=$(kubectl get pod -n $CODETRANS_NAMESPACE -l app=client-test -o jsonpath={.items..metadata.name})
+   echo "$CLIENT_POD"
+   accessUrl=$(kubectl get gmc -n $CODETRANS_NAMESPACE -o jsonpath="{.items[?(@.metadata.name=='codetrans')].status.accessUrl}")
+   kubectl exec "$CLIENT_POD" -n $CODETRANS_NAMESPACE -- curl $accessUrl  -X POST  -d '{"query":"    ### System: Please translate the following Golang codes into  Python codes.    ### Original codes:    '\'''\'''\''Golang    \npackage main\n\nimport \"fmt\"\nfunc main() {\n    fmt.Println(\"Hello, World!\");\n    '\'''\'''\''    ### Translated codes:"}' -H 'Content-Type: application/json' > $LOG_PATH/gmc_codetrans.log
```

Comment:
I just wonder why the test case is very different from codetrans mega e2e test. https://github.com/opea-project/GenAIExamples/blob/main/CodeTrans/tests/test_codetrans_on_gaudi.sh#L102.

Your test case is a microservice test, refer to: https://github.com/opea-project/GenAIExamples/blob/main/CodeTrans/tests/test_codetrans_on_gaudi.sh#L92

Reply:
yes, Daisy, you are making a good point, looking at it

Reply:
@daisy-ycguo pls check again